gym_bridges/envs/test_bridges_env.py

```diff
@@ -6,7 +6,17 @@
 from collections import defaultdict
 from parameterized import parameterized
 
-from bridges_env import BridgesEnv
+from gym_bridges.envs.bridges_env import BridgesEnv
+
+
+@pytest.mark.parametrize("width", [3, 9])
+def test_force_standard_config_render(width):
+    """The standard configuration only has ground at the bottom left
+    and right locations of the env."""
+    env = BridgesEnv(width=width, force_standard_config=True)
+    env.reset()
+    env.render(mode="pygame")
+    time.sleep(5)
```

Comment:
Prefer delete the sleep. I realize this is nice as a human to see it working. But now that's it's working, the main purpose of the test is that the code is not broken. If running in CI (in practice or in theory), no one is looking at the output.